packages/web-components/src/components/va-statement-of-truth/test/va-statement-of-truth.e2e.ts

```diff
@@ -73,7 +73,7 @@ describe('va-statement-of-truth', () => {
     const inputEl = await page.$('pierce/[name="veteran-signature"]');
     await inputEl.type('test');
     // wait for last event
-    await page.waitForTimeout(500);
+    await new Promise(resolve => setTimeout(resolve, 500));
```

Comment:
`waitForTimeout` is a deprecated method. Replacing with a simple promise resolve to achieve the same desired outcome.